src/Checkbox/Checkbox.less

```diff
@@ -2,22 +2,73 @@
 .moon-checkbox {
 	cursor: pointer;
 
+	.moon-formcheckbox-item &,
+	.moon-toggle-item & {
+		.moon-icon {
+			visibility: hidden;
+			background-color: transparent;
+		}
+		&[checked] .moon-icon {
+			visibility: visible;
+
+			.moon-neutral & {
+				background-color: transparent;
+			}
+		}
+	}
+
 	.moon-icon {
-		visibility: hidden;
 		margin: 0;
 		padding: 0;
+		width: 30px;
+		height: 30px;
```

Comment:
These would probably be better as a `@moon-checkbox-size` variable in the `variables.less` file